engine/console/game.go

```diff
@@ -212,18 +274,47 @@ func (game *Game) RunGame() {
 		}
 
 		// Draw Game state
-		drawInitialBox(s, boundary, boxStyle)
-		for _, snake := range game.snakes {
-			err := drawSnake(s, snake, boundary)
+		s.Clear()
+		if game.Over {
+			drawBox(s, boundary, boxStyle)
+			width, height := 12, 0
+			if game.Winner != -1 {
+				height = 4
+			} else {
+				height = 2
+			}
+			x1 := (boundary.BottomRight.X - boundary.TopLeft.X - width) / 2
+			y1 := (boundary.BottomRight.Y - boundary.TopLeft.Y - height) / 2
+			x2 := (boundary.BottomRight.X - boundary.TopLeft.X + width) / 2
+			y2 := (boundary.BottomRight.Y - boundary.TopLeft.Y + height) / 2
+			drawBox(s, Boundary{core.Coord{X: x1, Y: y1}, core.Coord{X: x2, Y: y2}}, blackBoxStyle)
+			drawText(s, x1 + 1, y1 + 1, x2 - 1, y2 - 1, blackBoxStyle, "Game Over")
+			if game.Winner != -1 {
+				text := "Winner " + strconv.Itoa(game.Winner)
+				drawText(s, x1 + 1, y1 + 3, x2 - 1, y2 - 1, blackBoxStyle, text)
+			}
+			s.Show()
+			continue
+		}
+		drawBox(s, boundary, boxStyle)
+		for id, snake := range game.Snakes {
+			if !snake.alive {
+				continue
+			}
+			err := drawSnake(s, id, snake, boundary)
 			if err != nil {
+				s.Fini()
 				log.Fatalf("%+v", err)
+				os.Exit(0)
 			}
 		}
-		for _, f := range game.food {
+		for _, f := range game.Food {
 			err := drawFood(s, f, foodStyle, boundary)
 			if err != nil {
-				println(err)
+				s.Fini()
+				fmt.Println(err)
 				log.Fatalf("%+v", err)
+				os.Exit(0)
 			}
 		}
 		s.Show()
```

Comment:
Seems it doesn't draw anything properly at all, if I try to actually wait a second between each ticks... We should investigate in the next task...